android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

```diff
@@ -189,6 +189,8 @@ private void handleResponse(HttpURLConnection connection) throws IOException
 					if (baseFile instanceof TiFile) {
 						responseFile = (TiFile) baseFile;
 					}
+				} else if (f instanceof TiFileProxy) {
+					responseFile = ((TiFileProxy) f).getBaseFile();
```

Comment:
This does not compile.
Member variable `responseFile` is of type `TiFile`, but the `getBaseFile()` method's return type is `TiBaseFile`.

The code should look something like the below, although it kind of duplicates the code above it.
```javascript
} else if (f instanceof TiFileProxy) {
	TiBaseFile baseFile = ((TiFileProxy) f).getBaseFile();
	if (baseFile instanceof TiFile) {
		responseFile = (TiFile) baseFile;
	}
}
```

Would you prefer that make the change?

Reply:
Mhh, thats weird. It did compile when using it in the first place. I just applied the change, thanks!